source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts

```diff
@@ -88,6 +88,11 @@ export class Route53ToApiGateway extends Construct {
     super(scope, id);
     defaults.CheckProps(props);
 
+    // Certificate must already be issued when the user is bringing an existing one
+    // If you are using privateHostedZoneProps, the certificate must already be issued
```

Comment:
Lots of comments of "if you are (something here), then you must have a certificate", but isn't our message on this construct "you must always provide a validated certificate", so these are actually misleading.